lib/components/user/places/place.js

```diff
@@ -0,0 +1,129 @@
+import { Button } from 'react-bootstrap'
+import { useIntl } from 'react-intl'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import { LinkContainerWithQuery } from '../../form/connected-links'
+import BaseIcon from '../../util/icon'
+
+const Container = styled.li`
+  align-items: stretch;
+  display: flex;
+`
+export const PlaceButton = styled(Button)`
+  background: none;
+  flex: 1 0 0;
+  overflow: hidden;
+  text-align: left;
+  text-overflow: ellipsis;
+`
+
+export const PlaceDetail = styled.span``
```

Comment:
Why did you decide to put these here instead of in the component's `styled.js`?

Reply:
To avoid a circular reference between styled.js and place.js

Reply:
That way also, styled.js defines different flavors of the `<Place>` component and its internal elements.